stl/src/xonce2.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <yvals_core.h>
+
+#include "awint.h"

```

Comment:
@BillyONeal , any reason to include whole `"awint.h"` instead of just `<synchapi.h>`?

Reply:
Ah, I see, you recovered what I added there. But apparently `"awint.h"` is superfluous now that there's no `GetProcAddress` etc. Will fix